src/hotspot/share/opto/memnode.cpp

```diff
@@ -427,23 +428,23 @@ Node *MemNode::Ideal_common(PhaseGVN *phase, bool can_reshape) {
 // Used by MemNode::find_previous_store to prove that the
 // control input of a memory operation predates (dominates)
 // an allocation it wants to look past.
-bool MemNode::all_controls_dominate(Node* dom, Node* sub) {
+Node::DomResult MemNode::all_controls_dominate(Node* dom, Node* sub) {
   if (dom == nullptr || dom->is_top() || sub == nullptr || sub->is_top())
-    return false; // Conservative answer for dead code
+    return DomResult::EncounteredDeadCode; // Conservative answer for dead code
```

Comment:
Please, fix style of this and other code you touched. Add `{}` for condition's body where it is missing.